guide/14-deep-learning/faster-rcnn-object-detector.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>We have seen how the one-shot object detection models, such as <a href="https://developers.arcgis.com/python/guide/how-ssd-works/" target="_blank">SSD</a>, <a href="https://developers.arcgis.com/python/guide/how-retinanet-works/" target="_blank">RetinaNet</a>, and YOLOv3, work. However, before the single-stage detectors were the norm, the most popular object detectors were from the multi-stage R-CNN family. First, there was R-CNN, then Fast R-CNN came along with some improvements, and then eventually, Faster R-CNN became the state-of-the-art multi-stage object detector. As is obvious from the names, these models evolved from one to the next, providing better performance at faster speeds. Although Faster R-CNN is not as fast as some of the later single-stage models, it remains one of the most accurate object detection models.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>